docs/faq.md

```diff
@@ -211,14 +212,57 @@ To limit liability and respect the DMCA, we only distribute patches that are app
 7. Click on the `Install` button.
 
 #### How do I install the latest version on Linux?  
+
+For a universal installation of dependencies (except AM2R 1.1) and the AM2RLauncher on all Linux Distributions, including the Steam Deck, you have the option to install the AM2RLauncher via Flatpak:
```

Comment:
I mostly agree with this, however I feel the Flatpak instructions should stay, as it shows an easy way for the user to install the program, and only takes two lines (Steps 3-5 can be removed). I can redirect distribution specific commands to the wiki, but keep in mind if we list dependencies here then they will have to be updated as well. 

Here's what I have made:

To install on Linux with no dependencies on every distribution, you can use the Universal method with Flatpak, otherwise you can install the program manually with dependencies. 
**Universal/Steam Deck**
1. Follow the instructions for your Distro to install [Flatpak](https://flatpak.org/setup/) if don't have it installed already. Steam Deck users can skip this step.
2. Install the [AM2RLauncher Flatpak](https://flathub.org/apps/details/io.github.am2r_community_developers.AM2RLauncher). This may be from the linked site, or your software center, such as `Discover` on the Steam Deck.
3. In the Launcher, click on the `Download` button and let it download the necessary patch files.
4. Click on the `Select 1.1` button and select your copy of AM2R 1.1 and let the AM2RLauncher verify it.
5. Click on the `Install` button.

**Manual Install**
A distribution specific installation of AM2R is available on the [Community Updates Releases](https://github.com/AM2R-Community-Developers/AM2RLauncher/releases) page, and requires you to download the following dependencies:
`.NET Runtime` (5.0 or higher), `xdelta3`, `gtk3`, `libappindicator3`, `webkitgtk`, `openssl`, `fuse2`, Multilib / 32-Bit architecture added, and the **32-Bit version** of `libpulse`.
A detailed installation for each distribution with commands is available on the [GitHub Wiki](https://github.com/AM2R-Community-Developers/AM2RLauncher/wiki/Installation-Process) page.

Tried to keep it as thin as possible for maintenance and not confusing the user.

Reply:
Ah sorry if i wasn't clear enough. I do want the flatpak mention to stay, just don't want to have a copy-pasted installation instruction. Something more akin to this
```
#### How do I install the latest version on Linux?  

###### Flatpak / Steam Deck
For a universal Linux install without the need to install any dependencies, or for Steam Deck users, you can choose to use the Flatpak version of the AM2RLauncher.
1. Follow the instructions for your Distro to install [Flatpak](https://flatpak.org/setup/) if don't have it installed already. Steam Deck users can skip this step.
2. Install the [AM2RLauncher Flatpak](https://flathub.org/apps/details/io.github.am2r_community_developers.AM2RLauncher). This may be from the linked site, or your software center, such as Discover on the Steam Deck.
3. Launch the AM2RLauncher, and follow the instructions from "Install Community Updates" further down.

First you need to download the right dependencies: `.NET Runtime` (6.0 or higher), `xdelta3`, `gtk3`, `libappindicator3`, `webkitgtk`, `openssl` and `fuse2`. In addition, you also need the following dependencies for AM2R itself: Multilib / 32-Bit architecture added, and the **32-Bit version** of `libpulse`.  
A detailed installation for each distribution with commands is available on the [GitHub Wiki](https://github.com/AM2R-Community-Developers/AM2RLauncher/wiki/Installation-Process) page.

After that, you need to download and start the AM2RLauncher:
1. Head to the AM2RLauncher download page, download the Linux version and unzip the file to its own folder: <https://github.com/AM2R-Community-Developers/AM2RLauncher/releases/latest>
2. Make sure that you have the following dependencies for the AM2RLauncher installed: 
3. Mark `AM2RLauncher.Gtk` as an executable, and then run it. This can be done like this: `chmod +x AM2RLauncher.Gtk` and `./AM2RLauncher`. Alternatively, if your desktop environment supports it, you can also double-click on the file after it has been marked as an executable.

###### Install Community Updates
1. Click on the `Download` button and let it download the necessary patch files.
2. Click on the `Select 1.1` button and select your copy of AM2R 1.1 and let the AM2RLauncher verify it.
3. Click on the `Install` button.
```

Reply:
Ah, my bad. I do think we should combine the flatpak and community updates install however. It being at the end may confuse some users who think the two middle paragraphs are needed for flatpak as well.

Reply:
The same logic applies the other way around: people who install manually (which currently is our biggest userbase) being confused on what to do after installing dependencies.
I think that having them seperated is the better approach, also for the cases where we may have different downloading methods.